gabbi/runner.py

```diff
@@ -136,8 +139,14 @@ def run_suite(handle, handler_objects, host, port, prefix, force_ssl=False,
         loader, name, data, data_dir, host, port, None, None, prefix=prefix,
         handlers=handler_objects, test_loader_name='gabbi-runner')
 
+    # The default runner stream is stderr.
+    stream = sys.stderr
+    if quiet:
+        # We want to swallow the output that the runner is
+        # producing.
```

Comment:
a) weird break but okay

b) your continued resistance to ternaries is kinda hilarious

Reply:
> b) your continued resistance to ternaries is kinda hilarious

I had a ternary there but changed it when I wanted the comments to be separate.